server_admin/topics/threat/auth-sessions-limit.adoc

```diff
@@ -50,11 +58,37 @@ ifeval::["{kc_dist}" == "wildfly"]
 </subsystem>
 ----
 
-Equivalent configuration using CLI commands:
+The equivalent configuration for the new map storage:
```

Comment:
I believe this is not necessary as the legacy distribution will be removed in next KC version.

Reply:
I see. I removed parts about Wildfly.